content/influxdb/cloud-iox/sql-reference.md

```diff
@@ -0,0 +1,596 @@
+
+```sql
+SELECT MEAN("water_level"), "location"
+FROM "h2o_feet" 
+GROUP BY "location","time"
+```
+
+### The HAVING clause
+
+ Use the `HAVING` clause to filter query results based on a specified condition. The `HAVING` clause must follow the `GROUP BY` clause but precedes the `ORDER BY` clause.
+
+#### Examples
+
+```sql
+SELECT MEAN("water_level"), "location", "time"
+FROM "h2o_feet" 
+GROUP BY "location","time"
+HAVING MEAN("water_level") > 4
+
+SELECT MEAN("water_level"), "location", "time"
+FROM "h2o_feet" 
+GROUP BY "location","time"
+HAVING MEAN("water_level") > 9.8
+ORDER BY "time"
+```
+
+### The UNION clause
+
+The `UNION` clause combines the results of two or more SELECT statements without returning any duplicate rows. `UNION ALL` returns all results, including duplicates. 
+
+#### Examples
+
+```sql
+SELECT 'pH'
+FROM "h2o_pH"
+UNION ALL
+SELECT "location"
+FROM "h2o_quality"
+```
+
+### The ORDER BY clause 
+
+The `ORDER BY` clause orders results by the referenced expression.  The result order is `ASC` **by default**.  You can filter data based on fields, tags, and/or timestamps.
+
+#### Examples
+
+```sql
+SELECT "water_level" 
+FROM "h2o_feet" 
+WHERE "location" = 'coyote_creek'  
+ORDER BY time DESC
+```
+
+### The LIMIT clause
+
+The `LIMIT` clause limits the number of rows to be a maximum of count rows. The count should be a non-negative integer.
+
+#### Examples
+
+```sql
+SELECT "water_level","location" 
+FROM "h2o_feet" 
+LIMIT 10
+```
+
+### The WITH clause 
+
+The `WITH` clause provides a way to write auxiliary statements for use in a larger query.  It can help break down large, complicated queries into simpler forms. 
+
+```sql
+WITH summary_data as
+(SELECT degrees, location, time 
+  FROM average_temperature)
+SELECT * FROM summary_data
+```
+
+### The OVER clause 
+
+The `OVER` clause is used with SQL window functions. A `window function` performs a calculation across a set of table rows that are related in some way to the current row. While similar to aggregate functions, window functions output results into rows retaining their separate identities.   
+
+```sql
+SELECT time, water_level 
+FROM
+(SELECT time, "water_level", row_number() 
+ OVER (order by water_level desc) as rn 
+FROM h2o_feet) 
+WHERE rn <= 3;
+```
+
+## Comments
+
+Use comments to describe and add detail and notes to your queries.  
+
+ - Single line comments use the double hyphen `--` symbol. Single line comments end with a line break.
+ - Multi-line comments begin with `/*` and end with ` */`. Multi-line comments span multiple lines. 
+
+ ```sql
+ Single line comments:
+
+-- Examples
+
+SELECT COUNT("water_level") --no timestamp needed
+FROM "h2o_feet"
+
+Multiline comments:
+
+/* author:
+ * date:
+ */
+SELECT COUNT("water_level")
+FROM "h2o_feet"
+```
+
+## Schema information
+
+InfluxDB Cloud backed by InfluxDB IOx supports the following metedata schema queries:
+
+```sql
+SHOW tables
+
+SHOW columns FROM <measurement>
+```
+
+## Functions
```

Comment:
Do we not have another section specifically for functions? It's going to be a pain to maintain those and this section. I think we can just point users to the other function docs.

Reply:
we need a separate section, I started working on aggregates

I doubt this will be ready by Jan 31, but I can add after